ecleankernel/layout/blspec.py

```diff
@@ -149,6 +149,17 @@ def find_kernels(self,
                         ver, module_dict,
                         exclusions)
 
+                uki_map = {
```

Comment:
To be honest, now this looks like a needless complexity, given there's only one file here.

Reply:
I suppose, simplified it a bit now